src/types/steps.ts

```diff
@@ -59,6 +59,7 @@ export type StepOptionDocument = {
   photoCaptureFallback?: never // for cross-compatibility with StepOptionFace in withCrossDeviceWhenNoCamera
   showCountrySelection?: boolean
   useLiveDocumentCapture?: boolean
+  useMultiFrameCapture?: boolean
```

Comment:
Can you update the README to reflect this change?

Reply:
Done